src/pages/staking/index.js

```diff
@@ -175,251 +242,260 @@ const StakingPage = ({ data, location }) => {
         description={translateMessageId("page-staking-meta-description", intl)}
       />
       <PageHero content={heroContent} />
+      <StakingStatsBox />
       <Divider />
       <Content>
-        <Vision>
-          <Breadcrumbs slug={location.pathname} startDepth={1} />
-          <h2>
-            <Translation id="page-staking-just-staking" />
-          </h2>
-          <p>
-            <Translation id="page-staking-description" />{" "}
-            <Link to="/upgrades/beacon-chain/">
-              <Translation id="page-staking-the-beacon-chain" />
-            </Link>
-          </p>
-          <CardContainer>
-            {paths.map((path, idx) => (
-              <StyledCard
-                key={idx}
-                emoji={path.emoji}
-                title={path.title}
-                description={path.description}
-              >
-                {path.url && <Link to={path.url}>{path.link}</Link>}
-              </StyledCard>
-            ))}
-          </CardContainer>
-        </Vision>
+        <Breadcrumbs slug={location.pathname} startDepth={1} />
+        <h2>What is staking?</h2>
+        <p>
+          Staking is the act of locking up ETH to give you the right to
+          participate in block proposals on the network. Anyone who holds even a
+          small amount of ETH can consider staking.
+        </p>
```

Comment:
```suggestion
        </p>
        <p><Link to="/get-eth/">Learn how to get ETH</Link>.</p>
```
Perhaps a quick reference for folks who need ETH? 😄

Reply:
Oof... looking to stake and still no ETH? ngmi 😉 

hah, can put this in =)